app/javascript/lib/rich_text/unfurl/unfurler.js

```diff
@@ -24,7 +25,8 @@ export default class Unfurler {
   #addEventListeners() {
     addEventListener("trix-initialize", function(event) {
       if (this.#editorElementPermitsAttribute(event.target, "href")) {
-        return event.target.addEventListener("trix-paste", this.#didPaste.bind(this))
+        event.target.addEventListener("trix-paste", this.#didPaste.bind(this))
+        event.target.addEventListener("trix-change", this.#didChange.bind(this))
```

Comment:
We prefere is the events are bount from the HTML elements, this makes the controllers more reusable.